### DISCUSSION TITLE
Customise the AzuraCast

### DISCUSSION THREAD

**Discussion by freygabor** [AzuraCast/AzuraCast#3775] [Q&A]
Hello,

I want to customise the azuracast look both on front and backend, but most important the admin side, colors, logo..etc.
I installed it on the Ubuntu server and docker and I need to see the files via ftp.
I installed the vsftpd on the server and I can login but when i visit the var/azuracast folder I can see only 3x files (not folders) and no others like folders, www folders or anything.

So my question is where can i find the folders and files so i can work with the design and looks.

PS. I dont want to remove the footer..etc just i want to edit the files to get different colours, add more description to the footer and header...etc.

Many Thanks all help

**Comment by skeggia84** [comment]
Use CSS

**Reply by freygabor** [reply]
Thanks. Do you have a CSS code how I can add logo with link forward? So if someone click on then will go to my site.
I don't want to remove AzuraCast on top or the footer.

Also do you have any CSS code to change the tables colour to mine? 

Many thanks
